__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
```

Comment:
You could not deconstruct the seed data, it should be set up to deconstruct inside the function, meaning you can just require the whole object and pass that into it, for example `const data = require("...")`